exercises/E-conditionals/exercise.js

```diff
@@ -8,7 +8,11 @@
 
 var name = "Daniel";
 var danielsRole = "mentor";
-
+if ((danielsRole = "mentor")) {
```

Comment:
Careful here, you are using the `=` which is the operator to assign a value. I think you want to do a comparison here. What operator is used for comparison?